ddtrace/contrib/internal/asgi/middleware.py

```diff
@@ -533,18 +491,8 @@ def _handle_websocket_send_message(self, scope: Mapping[str, Any], message: Mapp
             call_trace=False,
             activate=True,
             tags={COMPONENT: self.integration_config.integration_name, SPAN_KIND: SpanKind.PRODUCER},
-        ) as ctx, ctx.span as send_span:
-            # Propagate context from the handshake span (baggage, origin, sampling decision)
-            # set tags related to peer.hostname
-            _set_client_ip_tags(scope, send_span)
-
-            send_span.set_link(
-                trace_id=request_span.trace_id,
-                span_id=request_span.span_id,
-                attributes={SPAN_LINK_KIND: SpanLinkKind.RESUMING},
-            )
-            send_span.set_metric(websocket.MESSAGE_FRAMES, 1)  # PERF: avoid setting via Span.set_metric
-            _set_message_tags_on_span(send_span, message)
+        ) as ctx, ctx.span as send_span:  # noqa: F841
```

Comment:
same here, if you add this span name to the list for "_finish_span" events in _trace_handlers.py you can get rid of the `, ctx.span as send_span` from this.